src/Common/src/Interop/Windows/Kernel32/Interop.GetCPInfoEx.cs

```diff
@@ -19,14 +19,14 @@ private unsafe struct CPINFOEXW
             internal fixed byte LeadByte[12];
             internal char UnicodeDefaultChar;
             internal uint CodePage;
-            internal fixed byte CodePageName[260];
+            internal fixed char CodePageName[260];
```

Comment:
This is the actual bug

Reply:
`internal fixed char CodePageName[MAX_PATH];` ?

Reply:
@jkotas, how did you find this? Is a test failing? Something getting corrupted? Crash?

Reply:
I wanted to see what the `CodePageName` returned by the OS is. I copied this interop definition from CoreFX to my local test and the string created from `byte CodePageName` was garbage. It is how I have noticed the bug.